Core/CKAN-core.csproj

```diff
@@ -1,159 +1,64 @@
-    <Compile Include="Net\NetFileCache.cs" />
-    <Compile Include="Net\NetModuleCache.cs" />
-    <Compile Include="Net\Repo.cs" />
-    <Compile Include="Platform.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Registry\AvailableModule.cs" />
-    <Compile Include="Registry\InstalledModule.cs" />
-    <Compile Include="Registry\IRegistryQuerier.cs" />
-    <Compile Include="Registry\Registry.cs" />
-    <Compile Include="Registry\RegistryManager.cs" />
-    <Compile Include="Relationships\RelationshipResolver.cs" />
-    <Compile Include="Relationships\SanityChecker.cs" />
-    <Compile Include="ServiceLocator.cs" />
-    <Compile Include="Types\CkanModule.cs" />
-    <Compile Include="Versioning\CkanModuleVersion.cs" />
-    <Compile Include="Versioning\UnmanagedModuleVersion.cs" />
-    <Compile Include="Types\ExportFileType.cs" />
-    <Compile Include="Types\FileType.cs" />
-    <Compile Include="Types\GameComparator\BaseGameComparator.cs" />
-    <Compile Include="Types\GameComparator\GrasGameComparator.cs" />
-    <Compile Include="Types\GameComparator\IGameComparator.cs" />
-    <Compile Include="Types\GameComparator\StrictGameComparator.cs" />
-    <Compile Include="Types\GameComparator\YoyoGameComparator.cs" />
-    <Compile Include="Types\Kraken.cs" />
-    <Compile Include="Types\License.cs" />
-    <Compile Include="Types\ModuleInstallDescriptor.cs" />
-    <Compile Include="Types\ModuleResolution.cs" />
-    <Compile Include="Versioning\ProvidesModuleVersion.cs" />
-    <Compile Include="Types\ReleaseStatus.cs" />
-    <Compile Include="Types\Repository.cs" />
-    <Compile Include="Versioning\ModuleVersion.cs" />
-    <Compile Include="User.cs" />
-    <Compile Include="VersionFormat.cs" />
-    <Compile Include="Versioning\KspVersion.cs" />
-    <Compile Include="Versioning\KspVersionBound.cs" />
-    <Compile Include="Versioning\KspVersionCriteria.cs" />
-    <Compile Include="Versioning\KspVersionRange.cs" />
-    <Compile Include="Win32Registry.cs" />
-    <Compile Include="Utilities.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
+  
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Autofac" Version="4.9.2" />
+    <PackageReference Include="ICSharpCode.SharpZipLib.Patched" Version="0.86.5.1" />
+    <PackageReference Include="log4net" Version="2.0.8" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="TxFileManager.NETStandard" Version="2.0.0" />
+    <PackageReference Include="Shaman.CurlSharp" Version="0.6.3.1" />
+    <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
   </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="builds.json" />
```

Comment:
We need this embedded resource.

Reply:
Ah! That explains why netkan stopped working.

Reply:
:+1: the new commit fixed it.